### ISSUE TITLE
An HTML helper for srcset

### ISSUE THREAD

**Issue by tmjoen** [twined/brando#8]
Include the picture polyfill.

**Comment by tmjoen** [comment]
Ideas:

1. Create a srcset registry as an optional key in our `Brando.Type.ImageConfig` type. Here we can supply a map in the same way we do with `sizes`, and map which `size` key the srcset entry should reference.

```
  has_image_field :avatar, %{
    allowed_mimetypes: ["image/jpeg", "image/png"],
    default_size: :medium,
    upload_path: Path.join("images", "avatars"),
    random_filename: true,
    size_limit: 10240000,
    sizes: %{
      "micro"  => %{"size" => "25x25", "quality" => 100, "crop" => true},
      "thumb"  => %{"size" => "150x150", "quality" => 100, "crop" => true},
      "small"  => %{"size" => "300", "quality" => 100},
      "medium" => %{"size" => "500", "quality" => 100},
      "large"  => %{"size" => "700", "quality" => 100},
      "xlarge" => %{"size" => "900", "quality" => 100}
    },
    srcset: %{
       "small" => "300w",
       "medium" => "500w",
       "large" => "700w"
    }
  }

**Comment by tmjoen** [comment]
Implemented in 431b1667c5c19a4596199f0e1416d892f115f7b8
